### ISSUE TITLE
Búa til texta/ritgerð/script fyrir síðuna

### ISSUE THREAD

**Issue by davidercool** [davidercool/VSH-Hopverkefni#1]
Ég panta að gera þetta ekki

**Comment by davidercool** [comment]
Ok Ísar tekur að sér að gera Lore partinn

**Comment by IsarDadi** [comment]
Ég get gert nokkuð rough summary af öllum expansion pökkum frá leiknum og legion

**Comment by davidercool** [comment]
ok! Ég get mögulega skrifað texta um sögu leiksins (ekki lore/innihald leiksins)
En Andri þarf að finna sér eitthvað að gera

**Comment by IsarDadi** [comment]
Meinarðu bara hvernig hann hefur þróast frá því að hann var gerður ? Heads up, Mæti ekki í tíma á morgun, er veikur og þarf að sjá hvort að mér batni fyrir tölvutækni

**Comment by davidercool** [comment]
Pretty much, byrja á því að segja smá frá blizzard og svo hvernig þessi sería byrjaði með warcraft 1, 2 og 3. og hvenær öll release voru þannig dót

**Comment by IsarDadi** [comment]
Good idea, Ég byrja að gera lore og gameplay mechanics á morgun og Andri getur byrjað að koma með hugmyndir fyrir Uppsetningu á HTML og CSS útlitið

**Comment by davidercool** [comment]
ok! ég er með hugmynd um hvernig síðan sem þú ert að skrifa textann fyrir gæti litið út

**Comment by IsarDadi** [comment]
Ok, gætir þú sagt mér frá henni á fimmtudaginn í þar næsta vefsíðuhönnunartíma ?

**Comment by davidercool** [comment]
ekki á morgun?

**Comment by IsarDadi** [comment]
Kem ekki á morgun

**Comment by davidercool** [comment]
ok

**Comment by IsarDadi** [comment]
Ég gerði aðeins yfir 1000 orð fyrir þetta og ég myndi halda að það væri nóg, ég get copy pasteað þetta hingað
